### PR TITLE
Governance prerequisites

### PR DESCRIPTION
**Pull request by neildawson**
Changes to governance proposal tutorials, seeking to:

* Improve clarity around what you need to make a proposal
* Reduce repetition, or centralise repeated content
* Integrate all useful network parameters

Changes include:

* Mentioning typical requirements on the governance proposals index page, priming visitors to expect further requirements
* Replacing the "Proposing: What you need to know" section with a short "Requirements" section at the start of each tutorial, which is easier to scan and harder to miss
* Moving the submission instructions into the tabs with the associated templates, so you only see the instructions relevant to your chosen method
* Centralising those submission instructions in components, so they can all be edited in one place
* Creating a new "Voting and enacting" section at the end of each tutorial, where we can mention important thresholds and reinforce the required off-chain activities
* Moved field descriptions from lists to tables, so they're easier to scan, and changed the title from "Anatomy[...]" to "Fields" for consistency with other titles

Closees #112

### CODE REVIEW

**Comment by neildawson** [comment]
@candida-d I made the changes we discussed on Friday:

* Changed "Voting and enacting" to "Voting and enactment"
* Stated explicitly when the proposal will be enacted if the vote is passed
* Linked the mention of participation to an explanation in concepts
* Changed the "Fields" headings back to "Anatomy of…"
* Added the list of fields that cannot be changed in an update market proposal
* Removed Telegram links

You mentioned you'd want to do some copy tweaks, is there anything else I can do to finalise this PR?

File: docs/tutorials/proposals/freeform-proposal.md [hunk 1]
```diff
@@ -20,21 +22,44 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Create a freeform proposal
-The aim of this is to allow community to provide votes on proposals which don't change any of the behaviour of the currently running Vega blockchain. That is to say, at enactment time, no changes are effected on the system, but the record of how token holders voted will be stored on chain. 
+Freeform proposals allow the community to vote on matters which don't change any of the behaviour of the currently running Vega blockchain.
```
Comment (candida-d) [inline docs/tutorials/proposals/freeform-proposal.md:25]:
```suggestion
Freeform proposals allow the community to propose and vote on matters which don't change any of the behaviour of the currently running Vega blockchain.
```

File: docs/tutorials/proposals/freeform-proposal.md [hunk 2]
```diff
@@ -20,21 +22,44 @@ import Tabs from '@theme/Tabs';
...
 
-Note that to create a freeform proposal you need <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minProposerBalance" formatter='governanceToken' hideName={true} suffix="tokens" />, while to vote you need <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minVoterBalance" suffix="tokens" formatter='governanceToken' hideName={true}/>.
+At enactment time, no changes are effected on the system, but the record of how token holders voted will be stored on chain. 
+
+## Requirements
+
+You will need:
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minProposerBalance" hideName={true} suffix="tokens"/> associated with that public key
```
Comment (candida-d) [inline docs/tutorials/proposals/freeform-proposal.md:34]:
```suggestion
- A minimum of whichever is larger, associated with that public key: <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minProposerBalance" hideName={true} suffix="tokens"/> or <NetworkParameter frontMatter={frontMatter} param="spam.protection.proposal.min.tokens" hideName={true} suffix="tokens"/>
```

File: docs/tutorials/proposals/freeform-proposal.md [hunk 3]
```diff
@@ -20,21 +22,44 @@ import Tabs from '@theme/Tabs';
...
-    <NewFreeformWin />
+  <TabItem value="win" label="Command line (Windows)">
+		<TerminalInstructions />
+		<NewFreeformWin />
   </TabItem>
 </Tabs>
+
+## Voting and enactment
+
+All proposals are voted on by the community. Community members need a minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minVoterBalance" suffix="tokens" hideName={true} /> to vote. Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```
Comment (candida-d) [inline docs/tutorials/proposals/freeform-proposal.md:61]:
```suggestion
All proposals are voted on by the community. To vote, community members need, at a minimum, the larger of <NetworkParameter frontMatter={frontMatter} param="spam.protection.voting.min.tokens" suffix="tokens" hideName={true} or <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minVoterBalance" suffix="tokens" hideName={true} />, associated to their Vega key. 

Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.requiredParticipation" formatter="percent" hideName={true} /> and a voting majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential.
```

File: docs/tutorials/proposals/index.md [hunk 4]
```diff
@@ -15,12 +15,21 @@ import Topic from '/docs/topics/_topic-governance.mdx'
...
 Governance with tokenholder participation is essential on Vega. Markets and assets can only be added through governance, and network parameters, markets and assets all must be amended through governance as well.
 
+## Requirements
+
+You must meet certain criteria before you can submit. Requirements may differ depending on the type of proposal.
+
+You will typically need:
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum amount of Vega tokens associated with that public key
```
Comment (candida-d) [inline docs/tutorials/proposals/index.md:25]:
```suggestion
- A minimum amount of Vega tokens associated with that public key. The amount depends on the proposal type
```

File: docs/tutorials/proposals/network-parameter-proposal.md [hunk 5]
```diff
@@ -21,40 +22,39 @@ Network parameters are a constant (or an array of constants) in the system, the
...
+		<TerminalInstructions />
+		<UpdateNetworkParameterWin />
   </TabItem>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+## Voting and enactment
+
+All proposals are voted on by the community. Community members need a minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.minVoterBalance" suffix="tokens" hideName={true} /> to vote. Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```
Comment (candida-d) [inline docs/tutorials/proposals/network-parameter-proposal.md:56]:
```suggestion
All proposals are voted on by the community. To vote, community members need, at a minimum, the larger of  <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.minVoterBalance" suffix="tokens" hideName={true} />, or <NetworkParameter frontMatter={frontMatter} param="spam.protection.voting.min.tokens" suffix="tokens" hideName={true} associated with their Vega key. 

Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```

File: docs/tutorials/proposals/new-asset-proposal.md [hunk 6]
```diff
@@ -39,40 +48,31 @@ When adding an ERC-20 asset to the bridge, the key details are compared to the s
...
-    <NewAssetWin />
+  <TabItem value="win" label="Command line (Windows)">
+		<TerminalInstructions />
+		<NewAssetWin />
   </TabItem>
 </Tabs>
+
+## Voting and enactment
+
+All proposals are voted on by the community. Community members need a minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.minVoterBalance" suffix="tokens" hideName={true} /> to vote. Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```
Comment (candida-d) [inline docs/tutorials/proposals/new-asset-proposal.md:74]:
```suggestion
All proposals are voted on by the community. To vote, community members need, at a minimum, the larger of  <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.minVoterBalance" suffix="tokens" hideName={true} /> or <NetworkParameter frontMatter={frontMatter} param="spam.protection.voting.min.tokens" suffix="tokens" hideName={true} associated with their Vega key. 

Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 7]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+
+You will need:
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerBalance" hideName={true} suffix="token"/> associated with that public key
+- A minimum equity-like share of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerEquityLikeShare" hideName={true} />
+- Familiarity with [governance on Vega](../../concepts/vega-protocol.md#governance), particularly off-chain activities
+
+## Anatomy of an update market proposal
+The `updateMarket` proposal is similar to the `newMarket` proposal, but most fields are not required. See the descriptions in the [new market proposal tutorial](new-market-proposal.md#fields) for more on each field.
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:34]:
```suggestion
The `updateMarket` proposal requires the same fields as a `newMarket` proposal. See the descriptions in the [new market proposal tutorial](new-market-proposal.md#fields) for more on each field.
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 8]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+### Locked fields
 
-## Update an existing market
-Participants with enough equity-like share of a market can propose changes to some market properties via governance using the `updateMarket` proposal. This is similar to the `newMarket` proposal, but most fields are not required.
+The following fields are immutable and cannot be changed.
 
-:::info
-Only participants with the minimum equity-like share can propose a change to a market. The minimum is defined by the network parameter `governance.proposal.updateMarket.minProposerEquityLikeShare`.
-:::
+- `marketId`
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:42]:
```suggestion
- `market ID`
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 9]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+- `decimalPlaces`
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:43]:
```suggestion
- `decimal places`
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 10]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+- `positionDecimalPlaces`
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:44]:
```suggestion
- `position decimal places`
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 11]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+- `name`
+- `settlementAsset`
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:46]:
```suggestion
- `settlement asset`
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 12]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerBalance" hideName={true} suffix="token"/> associated with that public key
+- A minimum equity-like share of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerEquityLikeShare" hideName={true} />
+- Familiarity with [governance on Vega](../../concepts/vega-protocol.md#governance), particularly off-chain activities
+
+## Anatomy of an update market proposal
+The `updateMarket` proposal is similar to the `newMarket` proposal, but most fields are not required. See the descriptions in the [new market proposal tutorial](new-market-proposal.md#fields) for more on each field.
+
+You must include all existing parameters from the original `newMarket` proposal, even if they are not being changed.
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:36]:
```suggestion
In addition to the parameters you want to change, you must include all existing parameters from the original `newMarket` proposal, even if they are not being changed.
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 13]
```diff
@@ -9,36 +9,76 @@ keywords:
...
-    <UpdateMarketWin />
+  <TabItem value="win" label="Command line (Windows)">
+		<TerminalInstructions />
+		<UpdateMarketWin />
   </TabItem>
 </Tabs>
+
+## Voting and enactment
+
+All proposals are voted on by the community. Community members need a minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minVoterBalance" suffix="tokens" hideName={true} /> to vote. Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:80]:
```suggestion
All proposals are voted on by the community. To vote, community members need, at a minimum, the larger of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minVoterBalance" suffix="tokens" hideName={true} /> or <NetworkParameter frontMatter={frontMatter} param="spam.protection.voting.min.tokens" suffix="tokens" hideName={true} associated to their Vega key. 

Your proposal will need [participation](../../concepts/vega-protocol#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```

File: docs/tutorials/proposals/network-parameter-proposal.md [hunk 14]
```diff
@@ -21,40 +22,39 @@ Network parameters are a constant (or an array of constants) in the system, the
...
-4. Your proposal will need enough voting weight to pass, so having community support of your proposal is essential.
-
-## Full proposal examples
-In the tabs below you'll see an annotated example, which describes what each field is for, a JSON example that can be used to submit on the token dApp, and command line examples for different operating systems. **You'll need to replace the example data with the relevant details before submitting.**
+## Requirements
+
+You will need:
 
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.minProposerBalance" hideName={true} suffix="token"/> associated with that public key
```
Comment (candida-d) [inline docs/tutorials/proposals/network-parameter-proposal.md:30]:
```suggestion
- A minimum of whichever is larger, associated with that public key: <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateNetParam.minProposerBalance" hideName={true} suffix="tokens"/> or <NetworkParameter frontMatter={frontMatter} param="spam.protection.proposal.min.tokens" hideName={true} suffix="tokens"/>
```

File: docs/tutorials/proposals/new-asset-proposal.md [hunk 15]
```diff
@@ -20,6 +21,14 @@ import TabItem from '@theme/TabItem';
 # Propose a new asset
 This page provides a tutorial for submitting a proposal for a new ERC-20 asset to be used as collateral. It describes what can be proposed, what you need to propose a new asset, and provides proposal templates that you will need to edit before submitting.
 
+## Requirements
+
+You will need:
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.minProposerBalance" hideName={true} suffix="token"/> associated with that public key
```
Comment (candida-d) [inline docs/tutorials/proposals/new-asset-proposal.md:29]:
```suggestion
- A minimum of whichever is larger, associated with that public key: <NetworkParameter frontMatter={frontMatter} param="governance.proposal.asset.minProposerBalance" hideName={true} suffix="tokens"/> or <NetworkParameter frontMatter={frontMatter} param="spam.protection.proposal.min.tokens" hideName={true} suffix="tokens"/>
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 16]
```diff
@@ -9,36 +9,76 @@ keywords:
...
 import TabItem from '@theme/TabItem';
 
+Make changes to an existing market.
+
+## Requirements
+
+You will need:
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerBalance" hideName={true} suffix="token"/> associated with that public key
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:29]:
```suggestion
- A minimum of whichever is larger, associated with that public key: <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerBalance" hideName={true} suffix="tokens"/> or <NetworkParameter frontMatter={frontMatter} param="spam.protection.proposal.min.tokens" hideName={true} suffix="tokens"/>
```

File: docs/tutorials/proposals/update-market-proposal.md [hunk 17]
```diff
@@ -9,36 +9,76 @@ keywords:
...
+- A minimum equity-like share of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerEquityLikeShare" hideName={true} />
```
Comment (candida-d) [inline docs/tutorials/proposals/update-market-proposal.md:30]:
```suggestion
- A minimum equity-like share in the market of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minProposerEquityLikeShare" hideName={true} />
```

**Review by candida-d — CHANGES_REQUESTED** [review]
Made some changes to add in spam protection parameters and fix a few other details

File: docs/tutorials/proposals/index.md (hunk 4 above)
Comment (neildawson) [reply docs/tutorials/proposals/index.md:25]: The sentence just above this says "Requirements may differ depending on the type of proposal". Should we say it there, or here?

Comment (candida-d) [reply docs/tutorials/proposals/index.md:25]: Oh yes, forgot about that. Just at the top is fine - if it turns out to be unclear it can move.

**Review by candida-d — APPROVED** [review]

**Pull request merged**